pkgs/development/tools/build-managers/mill/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "mill";
-  version = "0.9.3";
+  version = "0.9.5";
 
   src = fetchurl {
     url = "https://github.com/lihaoyi/mill/releases/download/${version}/${version}";
```

Repo owner is now `com-lihaoyi`, so needs a small change:

```suggestion
    url = "https://github.com/com-lihaoyi/mill/releases/download/${version}/${version}";
```

---

Good catch, thanks!